ts/src/api/services/demo.service.ts

```diff
@@ -24,13 +24,24 @@ export const deleteDemoItemService = async (paramsId: string) => {
   return query;
 }
 
-export const updateOneDemoItemPropertyValueService = async (paramsId: string, requestBody: { propName: string, value: string }[]) => {
-  const updateOps: Record<string, string> = {};
-  for (const ops of requestBody) {
-    updateOps[ops.propName] = ops.value;
-  }
-  const query = await Demo.updateOne({ _id: paramsId }, { $set: updateOps }).exec();
-  return query;
+// const testFn = (obj: DemoDocument, key: keyof DemoDocument, value: (any |never) ) => {
+//   obj[key] = value
+// }
```

Comment:
**Review comment 2:** Clean up this comments @samuko-things

Reply:
> **Review comment 2:** Clean up this comments @samuko-things

ok